app/lib/features/pins/widgets/pin_list_item.dart

```diff
@@ -50,31 +88,95 @@ class _PinListItemState extends ConsumerState<PinListItem> {
     }
 
     return InkWell(
-      onTap: onTap,
+      onTap: () => onTap(pin),
       onLongPress: openItem,
       child: Card(
-        child: ListTile(
-          key: Key(pin.eventIdStr()), // FIXME: causes crashes in ffigen
-          leading: Icon(isLink ? Atlas.link_chain_thin : Atlas.document_thin),
-          title: Text(
-            pin.title(),
-            softWrap: false,
-            overflow: TextOverflow.ellipsis,
-          ),
-          titleTextStyle: Theme.of(context).textTheme.titleSmall,
-          subtitle: widget.showSpace
-              ? Wrap(
-                  alignment: WrapAlignment.start,
-                  children: [SpaceChip(spaceId: spaceId)],
-                )
-              : null,
-          onTap: onTap,
-          trailing: IconButton(
-            icon: isLink
-                ? const Icon(Icons.open_in_full_sharp)
-                : const Icon(Icons.chevron_right_sharp),
-            onPressed: openItem,
-          ),
+        child: LayoutBuilder(
+          builder: (context, constraints) {
+            return Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 12),
+              child: Column(
+                key: Key(pinId), // FIXME: causes crashes in ffigen
```

Comment:
pretty sure this fixme isn't useful here anymore
```suggestion
                key: Key(pinId),
```